components/builder-protocol/src/originsrv.rs

```diff
@@ -37,10 +37,10 @@ impl Routable for OriginGet {
 }
 
 impl Routable for OriginCreate {
-    type H = InstaId;
+    type H = String;
 
     fn route_key(&self) -> Option<Self::H> {
-        Some(InstaId(self.get_owner_id()))
+        Some(self.get_name().to_string())
```

Comment:
I think this is a nice change. It'll mean that the natural key always points to the place where the Origin lives. This does mean it's more difficult to figure out _all origins_ for a particular user, though. We'll need to store that somewhere else, are we already?